Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM quay.io/prometheus/busybox:latest
 
 ADD .build/linux-amd64/operator /bin/operator
+ADD ./contrib/prometheus-config-reloader/prometheus-config-reloader /bin/prometheus-config-reloader
```

Comment:
I've thought of this as well, but I'm not sure I like it, especially if the container image repo is the same, then people will be maximum confused I think. Even today people think the operator runs as a sidecar. It also has a large impact on the container size, the reload container is very very small whereas operator+reloader is a pretty large image.

Reply:
:+1: